editor/assets/chunks/standard-surface-entry.chunk

```diff
@@ -43,16 +43,14 @@
   layout(location = 0) out vec4 fragColor0;                        \
   layout(location = 1) out vec4 fragColor1;                        \
   layout(location = 2) out vec4 fragColor2;                        \
-  layout(location = 3) out vec4 fragColor3;                        \
                                                                    \
   void main () {                                                   \
     StandardSurface s; surf(s);                                    \
     fragColor0 = s.albedo;                                         \
     vec3 position;                                                 \
     HIGHP_VALUE_FROM_STRUCT_DEFINED(position, s.position);         \
-    fragColor1 = vec4(position, s.roughness);                      \
-    fragColor2 = vec4(float32x3_to_oct(s.normal), s.roughness, s.metallic); \
-    fragColor3 = vec4(s.emissive, s.occlusion);                    \
+    fragColor1 = vec4(float32x3_to_oct(s.normal), s.roughness, s.metallic); \
+    fragColor2 = vec4(s.emissive, s.occlusion);                    \
```

Comment:
Please reorder the logic so that all fragColor is assigned sequentially. position can be updated before them